lib/build.api.js

```diff
@@ -213,26 +192,22 @@ class Build {
             fs.mkdirSync(destBase);
```

Comment:
@generalredneck sure enough this will break in the right circumstances. Made the same change to my code.